File: code_samples/python/v1.0.0/get_available_payment_methods.py
from tatrapayplus.client import TBPlusSDK

client = TBPlusSDK(
    "your-client-id",
    "your-client-secret",
)

# Get all available payment methods for this transaction
methods = client.get_available_payment_methods(currency_code="EUR", country_code="SK", total_amount=10)

File: code_samples/php/v1.0.0/get_available_payment_methods.php
<?php
use Tatrapayplus\TatrapayplusApiClient\Api\TatraPayPlusAPIApi;

$tatrapayplus_api = new TatraPayPlusAPIApi(
    "your-client-id",
    "your-client-secret",
);

$available_methods = $tatrapayplus_api->getAvailableMethods(
    total_amount: 100,
    currency: "EUR",
    country: "SK"
);